src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -165,11 +206,20 @@ class MoveModal extends React.Component {
 }
 
 MoveModal.propTypes = {
-  entries: PropTypes.array
+  client: PropTypes.object.isRequired,
+  displayedFolder: PropTypes.object.isRequired,
+  entries: PropTypes.array,
+  t: PropTypes.func.isRequired,
+  sharingState: PropTypes.object // to know how to handle conflicts
```

Comment:
Maybe we can be more explicit for this comment. It's pretty clear now, but in a few months " why on earth we need sharingState to handle conflicts? It doesn't make any sens" :D

Reply:
:+1: